### ISSUE TITLE
Cannot generate coverage report with output from Coverlet 2.5.1

### ISSUE THREAD

**Issue by lshearer** [fdesjardins/coverage-viewer#9]
Using the current most recent version of coverlet (via `coverlet.msbuild` NuGet package version 2.5.1) and coverage-viewer (version 0.3.0), an error is thrown when generating the coverage report:
```
/Users/lyleshearer/.npm/_npx/78321/lib/node_modules/coverage-viewer/lib/statistics.js:22
    line => line.stats.find(stat => stat.name === 'Hits').value !== 0
                                                         ^

TypeError: Cannot read property 'value' of undefined
```
This can be repro'd by cloning and running [this script](https://github.com/lshearer/coverlet/blob/coverage-viewer-issue/coverage-viewer-issue-repro.sh) on a fork of the coverlet repo.

This appears to be because the coverlet output format for `coverage.json` changed, perhaps when support for branch coverage was added. The format of the [test asset](https://github.com/fdesjardins/coverage-viewer/blob/eaa07bfe282917da5459d00f9e2c378d17ca3795/assets/coverage.json) is quite different than [the one generated during that repro test](https://github.com/lshearer/coverlet/blob/741e5e7357ca4cac93c06f83fc315754737b99bf/test/coverlet.core.tests/coverage.json).

**Comment by fdesjardins** [comment]
Interesting, thanks for the write up and repro script. Glancing at the new format, you're right it does look very different. I'll take a closer look at supporting the new format in the next couple days.

**Comment by fdesjardins** [comment]
Well, I took a look a couple days ago and it actually doesn't look too bad. The main issue is with the new lines format. I'll try to clean this up and cut a new release this week.

**Comment by fdesjardins** [comment]
@lshearer Sorry for the delay. I just published a new version `0.4.0` that should resolve this issue. Let me know if you still have any issues. Thanks again for the bug report.
